lib/masamune/schema/catalog.rb

```diff
@@ -83,7 +83,7 @@ def pop
 
     def initialize(environment)
       self.environment = environment
-      @stores   = Hash.new { |h,k| h[k] = Masamune::Schema::Store.new(type: k) }
+      @stores   = Hash.new { |h,k| h[k] = Masamune::Schema::Store.new(environment, type: k) }
```

Comment:
add reference to `environment` so classes in schema namespace can reference logger